Tests/Common/LoRaDeviceTwin.cs

```diff
@@ -80,40 +82,46 @@ public sealed record LoRaDesiredTwinProperties : LoRaTwinProperties
         public bool? AbpRelaxMode { get; init; }
         public uint? FCntUpStart { get; init; }
         public uint? FCntDownStart { get; init; }
-        public int? FCntResetCounter { get; init; }
+        public uint? FCntResetCounter { get; init; }
         public int? Rx1DROffset { get; init; }
         public DataRateIndex? Rx2DataRate { get; init; }
         public ReceiveWindowNumber? PreferredWindow { get; init; }
         public int? RxDelay { get; init; }
         public char? ClassType { get; init; }
```

Comment:
Shouldn't this use `LoRaDeviceClassType`?

Reply:
Valid point, I propagated the usage of the type to its callers as well (`SimulatedDevice`, `TestDeviceInfo` etc) as part of c6680ad02b397af5e045c9b9d57ed44c56b64d57. Much better now!